improver/psychrometric_calculations/psychrometric_calculations.py

```diff
@@ -164,321 +280,104 @@ def calculate_d_enthalpy_dt(mixing_ratio, specific_heat,
         denominator = consts.R_WATER_VAPOUR * temperature ** 2
         return numerator/denominator + specific_heat
 
-    @staticmethod
-    def saturation_vapour_pressure_goff_gratch(temperature):
```

Comment:
Moved to ancillary creation plugin that generates the SVP table - only place this is used.